nidata/multimodal/hcp/__init__.py

```diff
@@ -144,88 +145,265 @@ def get_subject_list(self, n_subjects=None):
-                      for direction in ['LR', 'RL']
-                      for fil in [
-                          '{subj_id}_3T_BIAS_32CH.nii.gz',
-                          '{subj_id}_3T_BIAS_BC.nii.gz',
-                          '{subj_id}_3T_rfMRI_{scan}_{direction}_SBRef.nii.gz',
-                          '{subj_id}_3T_rfMRI_{scan}_{direction}.nii.gz',
-                          '{subj_id}_3T_SpinEchoFieldMap_LR.nii.gz',
-                          '{subj_id}_3T_SpinEchoFieldMap_RL.nii.gz']]
+        if not process:
+            rest_path = '%s/unprocessed/3T' % subj_id
 
+            files += ['%s/rfMRI_REST1_LR/%s_3T_BIAS_32CH.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/%s_3T_BIAS_BC.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/%s_3T_rfMRI_REST1_LR_SBRef.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/%s_3T_rfMRI_REST1_LR.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/%s_3T_SpinEchoFieldMap_LR.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/%s_3T_SpinEchoFieldMap_RL.nii.gz' % (rest_path, subj_id)]
+            files += ['%s/rfMRI_REST1_LR/LINKED_DATA/PHYSIO/%s_3T_rfMRI_REST1_LR_Physio_log.txt' % (rest_path, subj_id)]
+
+            files += ['%s/release-notes/rfMRI_REST1_unproc.txt' % subj_id]
+            files += ['%s/release-notes/rfMRI_REST2_unproc.txt' % subj_id]
         else:
-            raise NotImplementedError("Cannot (yet!) fetch '%s' files" % (
-                volume_type))
+            rest_path = '%s/MNINonLinear/Results' % subj_id
+            files += ['%s/rfMRI_REST1_LR/brainmask_fs.2.nii.gz' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_Regressors_dt.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_Regressors.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_AbsoluteRMS.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_AbsoluteRMS_mean.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_RelativeRMS.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/Movement_RelativeRMS_mean.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas.dtseries.nii' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/rfMRI_REST1_LR_Jacobian.nii.gz' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/rfMRI_REST1_LR_SBRef.nii.gz' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/rfMRI_REST1_LR.nii.gz' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/rfMRI_REST1_LR_Physio_log.txt' % rest_path]
+            files += ['%s/rfMRI_REST1_LR/RibbonVolumeToSurfaceMapping/goodvoxels.nii.gz' % rest_path]
+
+            files += ['%s/release-notes/rfMRI_REST1_preproc.txt' % subj_id]
+            files += ['%s/release-notes/rfMRI_REST2_preproc.txt' % subj_id]
+        return files
+
+    def get_func_files(self, process, type, subj_id):
```

Comment:
Docstring. `type` is a reserved word; `task` is a better variable name. `get_task_files` is a better function name.